nuitka/plugins/standard/PlaywrightPlugin.py

```diff
@@ -0,0 +1,168 @@
+
+    def __init__(self, include_browsers):
+        self.include_browsers = list(include_browsers)
+        self.installed_browsers = self.get_Installed_Playwright_Browsers()
+
+        if "all" in self.include_browsers:
+            self.include_browsers = self.installed_browsers
+        elif "ffmpeg" not in self.include_browsers and any(
+            browser.startswith("chrom") for browser in self.include_browsers
+        ):
+            for browser in self.installed_browsers:
+                if "ffmpeg" in self.installed_browsers[browser].name:
+                    self.include_browsers.append(browser)
+                    self.info(
+                        "Including 'ffmpeg' for chromium-based browser. It is required by playwright."
+                    )
+                    break
+
+        for browser in self.include_browsers:
+            if browser not in self.installed_browsers:
+                self.sysexit(
+                    f"Error, requested to include browser '{browser}' that was not found, the list of installed ones is '{', '.join(self.installed_browsers.keys())}'."
+                )
+
+    @staticmethod
+    def isAlwaysEnabled():
+        return True
+
+    @staticmethod
+    def isRelevant():
+        return isStandaloneMode()
+
+    @classmethod
+    def addPluginCommandLineOptions(cls, group):
+        group.add_option(
+            "--playwright-include-browser",
+            action="append",
+            dest="include_browsers",
+            default=[],
+            help="""\
+            Playwright browser to include. Can be specified multiple times. use "all" to include all installed browsers.
+            """,
+        )
+
+    def _getPlaywrightInfo(self):
+        "Determine the path of the playwright module."
+        try:
+            info = self.queryRuntimeInformationMultiple(
+                info_name="playwright_info",
+                setup_codes="""
+from playwright import __file__ as playwright_file
+from os.path import dirname
+""",
+                values=(("playwright_file", "dirname(playwright_file)"),),
+            )
+        except NuitkaCalledProcessError as e:
+            self.debug("Exception during detection: %r" % e)
+            raise
+
+        if info is None:
+            self.sysexit("Error, it seems 'playwright' is not installed or broken.")
+
+        return info
+
+    def getRegistryDirectory(self):
+
+        env_defined = os.environ.get("PLAYWRIGHT_BROWSERS_PATH")
+        path_home = os.path.expanduser("~")
+        playwright_module_path = self._getPlaywrightInfo().playwright_file
+
+        result = os.path.join(
+            playwright_module_path, "driver", "package", ".local-browsers"
+        )
+        if os.path.exists(result) and next(os.scandir(result), False):
+            return result
+        elif env_defined == "0":
+            return result
+        elif env_defined:
+            result = os.path(env_defined)
+        else:
+            cache_directory = ""
```

Comment:
Somewhere a comment that we cannot query this from playwright should be present, with it being JS I think, otherwise people will always wonder that's this.

It looks as if though, this is basically appdirs, so why don't we use our on AppDirs module, which probably covers more OSes even.

Reply:
we want to cover the OS'es playwright supports, using appdirs and supporting OS'es that playwright itself doesn't feels finicky